test/system_SUITE.erl

```diff
@@ -276,6 +278,23 @@ basic_get(Config) ->
     get_and_assert_empty(Channel, Q),
     teardown(Connection, Channel).
 
+named_connection_direct(Config) ->
+    named_connection(Config).
+named_connection_network(Config) ->
+    named_connection(Config).
```

Comment:
You don't need those wrappers. Just put `named_connection` in the `COMMON_PARALLEL_TEST_CASES` list.